helper/imagemin.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+module.exports = function(gulp, plugins, config, name) { // eslint-disable-line func-names
+  // Return function that is executed inside of .pipe()
+  return () => {
+    const theme          = config.themes[name],
+          srcBase        = config.projectPath + theme.src,
+
+          // Look for any images in the web/images directory and minify them in place
+          imageDir       = srcBase + '/web/images/**/*',
+          minnedImageDir = srcBase + '/web/images/';
+
+    let imageMinConfig;
+
+    try {
+        // Checks for project specific config in dev/tools/frontools/config
+        imageMinConfig = require('../helper/config-loader')('imageMinConfig.json', plugins, config);
+    } catch (e) {
+        imageMinConfig = { "optimizationLevel": 3, "progressive": true, "interlaced": true };
```

Comment:
We don't need to do this try/catch, if you check in the `config-loader` you'll see it throws an error if the config isn't found.

We should create a .sample config in the `config/` directory with the recommended defaults, it'll get copied over to the project on `gulp setup` and if it's then manually removed the `config-loader` module will throw out the error.